src/main/java/me/mrCookieSlime/Slimefun/SlimefunPlugin.java

```diff
@@ -237,6 +238,7 @@ else if (getServer().getPluginManager().isPluginEnabled("CS-CoreLib")) {
             new FireworksListener(this);
             new WitherListener(this);
             new IronGolemListener(this);
+            new BeeWingListener(this);
```

Comment:
Bees/Honey blocks have only been introduced in 1.15. Slimefun however is for 1.13, 1.14 and 1.15. So you will need to do adequate checks to make these version-exclusive...